src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: #000;
+  --hover-color: #00acdc;
+  --header-background-color: #fff;
+}
+
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' isn't available. For example, 'font-family: Roboto, Arial, sans-serif;' provides a default system font as a backup.